homeassistant/components/philips_js/config_flow.py

```diff
@@ -38,7 +50,9 @@ class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
     VERSION = 1
     CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_POLL
 
-    _default = {}
+    _current = {}
```

Comment:
Please don't set mutable objects as class attributes unless it's specifically needed. Just make an init method and set the attribute.